src/controllers/api/thoughts.js

```diff
@@ -0,0 +1,144 @@
+const { Thought, User } = require("../../models");
+
+const getAllThoughts = async (req, res) => {
+  try {
+    const thoughts = await Thought.find({});
+
+    return res.json({ data: thoughts });
+  } catch (error) {
+    console.log(`[ERROR]: Failed to get thoughts | ${error.message}`);
+
+    return res.status(500).json({
+      success: false,
+      error: "Failed to get thoughts",
+    });
+  }
+};
+
+const getThoughtById = async (req, res) => {
+  try {
+    const { id } = req.params;
+
+    const thought = await Thought.findById(id);
+
+    if (!thought) {
```

Comment:
Better ! This is what i would expect to see with the unhappy condition living in that if block